.devcontainer/devcontainer.json

```diff
@@ -77,5 +77,8 @@
     // Use 'postCreateCommand' to run commands after the container is created.
     // "postCreateCommand": "pip3 install --user -r requirements.txt",
     // Comment out connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
+    "mounts": [
```

Comment:
what is it exactly mounting?

Reply:
It is mounting our local file system to dev container, I thought it will be useful for testing on Jupyter notebook.